src/coreclr/vm/fcall.h

```diff
@@ -69,23 +23,10 @@
 // you can defer the setup of the frame to an unlikely code path (exception path)
```

Comment:
This paragraph should be delete - it is no longer possible to set up the frame in FCall